File: Plugins/Aspose_Cells_Java_for_PHP/src/aspose/cells/WorkingWithFiles/UtilityFeatures/ConvertingToMhtmlFiles.php
<?php


namespace Aspose\Cells\WorkingWithFiles\UtilityFeatures;

use com\aspose\cells\Workbook as Workbook;
use com\aspose\cells\HtmlSaveOptions as HtmlSaveOptions;
use com\aspose\cells\SaveFormat as SaveFormat;
//use com\aspose\cells\examples\Utils as Utils;



class ConvertingToMhtmlFiles {

    public static function run($dataDir=null)
    {

        $sveFormat = new SaveFormat();

        //Specify the file path
        $filePath = $dataDir . "Book1.xlsx";

        //Specify the HTML saving options
        $sv = new HtmlSaveOptions($sveFormat->M_HTML);

        //Instantiate a workbook and open the template XLSX file
        $wb = new Workbook($filePath);

        //Save the MHT file
        $wb->save($filePath . ".out.mht", $sv);

        // Print message
        print "<br>";
        print "<br>";
        print "Excel to MHTML conversion performed successfully.";
    }

}

File: Examples/src/main/java/com/aspose/cells/examples/loading_saving/ConvertingToMHTMLFiles.java
package com.aspose.cells.examples.loading_saving;

import com.aspose.cells.HtmlSaveOptions;
import com.aspose.cells.SaveFormat;
import com.aspose.cells.Workbook;
import com.aspose.cells.examples.Utils;

public class ConvertingToMHTMLFiles {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(ConvertingToMHTMLFiles.class) + "loading_saving/";

		// Specify the file path
		String filePath = dataDir + "Book1.xlsx";

		// Specify the HTML saving options
		HtmlSaveOptions sv = new HtmlSaveOptions(SaveFormat.M_HTML);

		// Instantiate a workbook and open the template XLSX file
		Workbook wb = new Workbook(filePath);

		// Save the MHT file
		wb.save(dataDir + "CToMHTMLFiles_out.mht", sv);

		// Print message
		System.out.println("Excel to MHTML conversion performed successfully.");

	}
}
